doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -145,31 +126,55 @@ Programmable proof search
    .. seealso:: :ref:`The Hints Databases for auto and eauto <thehintsdatabasesforautoandeauto>`
 
 
-.. tacn:: autounfold with {+ @ident}
-   :name: autounfold
+.. tacn:: autounfold {? @hintbases } {? @clause_dft_concl }
 
-   This tactic unfolds constants that were declared through a :cmd:`Hint Unfold`
-   in the given databases.
+   .. insertprodn clause_dft_concl hypident
+
+   .. prodn::
+      clause_dft_concl ::= in @in_clause
+      | {? at @occs_nums }
+      in_clause ::= * {? at @occs_nums }
+      | * %|- {? @concl_occ }
+      | {*, @hypident_occ } {? %|- {? @concl_occ } }
+      concl_occ ::= * {? at @occs_nums }
+      hypident_occ ::= @hypident {? at @occs_nums }
+      hypident ::= @ident
+      | ( type of @ident )
+      | ( value of @ident )
 
-.. tacv:: autounfold with {+ @ident} in @goal_occurrences
+   .. todo PR: move occs_num here from the rewriting chapter?  clause_dft_concl is used a lot BTW
 
-   Performs the unfolding in the given clause (:token:`goal_occurrences`).
+   :n:`in @in_clause`
+     Performs the unfolding in the given clause.
 
-.. tacv:: autounfold with *
+   :n:`with *`
+     Use the unfolding hints in all existing hint databases.
 
-   Uses the unfold hints declared in all the hint databases.
+   :n:`with {+ @ident }`
+     Use the unfolding hints in :n:`{+ @ident }` in addition to the database ``core``.
+     Use the fake database `nocore` to omit `core`.
```

Comment:
Yeah, we should hyperlink to a common description for commonly used bits of grammar rather than repeating the same text over and over.  That may be easier to do as a cleanup once the syntax has been updated everywhere.